app/_kong_plugins/response-ratelimiting/examples/response-rate-limit-with-local-policy.yaml

```
@@ -0,0 +1,19 @@
+description: |
+  Configure a custom response limit named `example_limit_name` with 10 responses allowed per minute.
+
+title: 'Response rate limit with local policy'
+
+weight: 900
+
+config:
+  limits:
+    example_limit_name:
```

```suggestion
    example-limit-name:
```
When I tested the underscores were replaced by dashes in the header name, so it might make sense to use that directly